rtrlib/transport/tcp/tcp_transport.c

```diff
@@ -192,7 +193,7 @@ const char *tr_tcp_ident(void *socket)
     return sock->ident;
 }
 
-int tr_tcp_init(const struct tr_tcp_config *config, struct tr_socket *socket)
+RTRLIB_EXPORT int tr_tcp_init(const struct tr_tcp_config *config, struct tr_socket *socket)
```

Comment:
this should (ideally) not be public

Reply:
Just like tr_ssh_init it must be, with the current api.

Reply:
yeah, I know - hence _ideally_ 😉 

so leave as is